### DISCUSSION TITLE
Ability to define configuration declaratively using Policies?

### DISCUSSION THREAD

**Discussion by johnhamelink** [mbnuqw/sidebery#1681] [Ideas]
I have been customising my Firefox configuration declaratively using Nix, and I'm using a custom UserChrome.css which makes use of Sidebery to provide a vertical tab UI. It'd be very nice indeed if I could define the config for this extension using a policy, because then I could include it in my nix configuration.

It is already possible to configure ublock origin this was as can be seen here: https://discourse.nixos.org/t/declare-firefox-extensions-and-settings/36265/26

This thread goes into detail about how ublock origin does this: https://github.com/libredirect/browser_extension/issues/905#issuecomment-2014867922

If this feature was enabled, then it'd be possible to use nix to set the configuration declaratively - nothing else would be required.

**Comment by cardboardcpu** [comment]
There's another unanswered thread looking for the same thing: https://github.com/mbnuqw/sidebery/discussions/2338.
Linking it here just in case.
